sections/aws.zsh

```diff
@@ -13,6 +13,7 @@ SPACESHIP_AWS_PREFIX="${SPACESHIP_AWS_PREFIX="using "}"
 SPACESHIP_AWS_SUFFIX="${SPACESHIP_AWS_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
 SPACESHIP_AWS_SYMBOL="${SPACESHIP_AWS_SYMBOL="☁️ "}"
 SPACESHIP_AWS_COLOR="${SPACESHIP_AWS_COLOR="208"}"
+SPACESHIP_AWS_PROFILE="${AWS_PROFILE="$AWS_VAULT"}"
```

Comment:
This should be inside the function, its not a config section

Reply:
If this variables isn't user configurable, It's better to move it inside function.